packages/termux-tools/pkg

```diff
@@ -1,11 +1,17 @@
 #!/bin/bash
 set -eu
 
-USER_AGENT='Termux-PKG/1.0 mirror-checker (termux-tools @PACKAGE_VERSION@) Termux (@TERMUX_APP_PACKAGE@; install-prefix:@TERMUX_PREFIX@)'
+if [ "$TERMUX_MAIN_PACKAGE_FORMAT" = "debian" ]; then
+	USER_AGENT='Termux-PKG/1.0 mirror-checker (termux-tools @PACKAGE_VERSION@) Termux (@TERMUX_APP_PACKAGE@; install-prefix:@TERMUX_PREFIX@)'
+fi
 
 show_help() {
 	local cache_size
-	cache_size=$(du -sh @TERMUX_CACHE_DIR@/apt/archives 2>/dev/null | cut -f1)
+	if [ "$TERMUX_MAIN_PACKAGE_FORMAT" = "debian" ]; then
+		cache_size=$(du -sh @TERMUX_CACHE_DIR@/apt/archives 2>/dev/null | cut -f1)
+	else
+		cache_size=$(du -sh @TERMUX_PREFIX@/var/cache/pacman/pkg 2>/dev/null | cut -f1)
+	fi
```

Comment:
It would always be good to catch the unexpected in order to prevent undefined behaviour